templates/field/field--text-with-summary--node.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
+/**
+ * @file
+ * Theme override for the text with summary fields used in nodes.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+<div class="ecl-editor">
+  {%- include 'field--bare.html.twig' -%}
+</div>
```

Comment:
If we do this, then every field of type text with summary will be printed without label. This is true for all the rest too.